ast.ts

```diff
@@ -1,5 +1,7 @@
 // import { TypeCheckError } from "./type-check";
 
+import { createModuleResolutionCache } from "typescript"
```

Comment:
Accidental import?

Reply:
We will check it out. I think it is added accidentally.